index.html

```diff
@@ -1256,31 +1344,78 @@
   // handle gUM error here
 });
 
+// ... initialize WebGL2 context when application starts.
+var gl = canvas.getContext("webgl2");
+// Activate the extension to use single component R32F texture format.
+gl.getExtension('EXT_color_buffer_float');
+
 // ... later, in the rendering loop ...
+gl.bindTexture(gl.TEXTURE_2D, depthTexture);
 gl.texImage2D(
    gl.TEXTURE_2D,
    0,
-   gl.RGB,
-   gl.RGB,
-   gl.UNSIGNED_BYTE,
-   depthVideo
-);
+   gl.R32F,
+   gl.RED,
+   gl.FLOAT,
+   depthVideo);
 
-&lt;script id="fragment-shader" type="x-shader/x-fragment"&gt;
-  varying vec2 v_texCoord;
-  // u_tex points to the texture unit containing the depth texture.
-  uniform sampler2D u_tex;
-  uniform float far;
-  uniform float near;
-  void main() {
-    vec4 floatColor = texture2D(u_tex, v_texCoord);
-    float dn = floatColor.r;
-    float depth = 0.;
-    depth = far * near / ( far - dn * ( far - near));
-    // ...
-  }
-&lt;/script&gt;
 </pre>
+      <h3>
+        WebGL2: <dfn>readPixels from float</dfn> texture
+      </h3>
+      <p>
+        This example extends <a>upload to float texture</a> example.
+      </p>
+      <pre class="example">
+// This code sets up a named framebuffer, attach the texture we upload the depth
+// video to as color attachment and, in rendering time, reads the texture
+// content  to Float32Array.
+
+// ... initialize texture and framebuffer for reading back the texture...
```

Comment:
Don't have a clue. It was there before.
I am often comfortable accepting existing code approach and replicating it without understanding what is it for.